### ISSUE TITLE
Get partially selected options

### ISSUE THREAD

**Issue by Owumaro** [riophae/vue-treeselect#70]
This is a feature request.

First of all, thanks for this plugin. So far I found options for everything I needed. Then I had this issue.

### Description

In non-flat & multi-select mode, I need to get "partially" selected options.
When using `value-consists-of="ALL"`, you only get selected items.

### Example

![image](https://user-images.githubusercontent.com/5205295/39308052-c769ee7a-4964-11e8-9ea2-0c100254a388.png)
The value is (with value-consists-of set to ALL):
```
[
  "team-ii",
  "person-e",
  "person-c",
  "person-d",
  "person-a"
]
```

I need:
```
[
  "company",
  "team-i",
  "person-a",
  "team-ii",
  "person-c",
  "person-d",
  "person-e"
]
```

### Use case

I have a table displaying statistics based on selected options, and I need to display parent options (with a partial total) even if all of its children aren't selected.

### Suggestions

The easiest solution would be to add a 4th value for value-consists-of, which would be wider than "ALL"... :smile:

Another approach would be to have a second value with partially selected options. Not sure if it's really useful to separate truly and partially selected options.

Any thoughts on this ?
Thanks !

**Comment by riophae** [comment]
Hi! How about adding a new option for `valueConsistsOf`, maybe `"ALL+"`?

**Comment by riophae** [comment]
Landed in v0.0.25.
